[VSTOOLS] Update Perquisites for VS2022

manishtr opened the pull request:
Signed-off-by: manish.r <manish.r@samsung.com>

### Change Description ###

Update Perquisites for VS2022

sangwook96 reviewed (approved):
Looks good to me

asm-foysal commented on docs/application/vstools/install.md:
```diff
@@ -18,7 +18,10 @@ To work with Visual Studio Tools for Tizen, your computer must have:
 - Visual Studio 2019 to use Tizen 4.0 and higher
-
+- Visual Studio 2022 to use Tizen 6.5 and higher
+- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Please make sure the same is installed through tizen package manager.
```
```suggestion
- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Please make sure the same is installed through Tizen package manager.
```

asm-foysal commented on docs/application/vstools/install.md:
```diff
@@ -18,7 +18,10 @@ To work with Visual Studio Tools for Tizen, your computer must have:
-
+- Visual Studio 2022 to use Tizen 6.5 and higher
+- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Please make sure the same is installed through tizen package manager.
+- Make sure to set the Tool Path (Tizen SDK) in Tools > Options > Tizen > Tools with installed Tizen Studio path. Also ensure to set the Google Chrome Path for Tizen Web App Debugging support.
```
```suggestion
- Make sure to set the Tool Path (Tizen SDK) in Tools > Options > Tizen > Tools with installed Tizen Studio path. Also, ensure to set the Google Chrome Path for Tizen Web App Debugging support.
```

asm-foysal reviewed (commented):
@manishtr there are some very minor corrections.

manishtr commented:
> @manishtr there are some very minor corrections.

Thanks for the review @asm-foysal , i have incorporated your suggestions

asm-foysal reviewed (approved):
@manishtr thank you for incorporating all the changes.

@mijin85cho I'm approving this PR for merging.

mijin85cho commented:
@asm-foysal Thanks for the review.

Just one thing to share - It is kind of style issue, but using "please" is not recommended.

**Reference :** https://developers.google.com/style/word-list#letter-p

> Don't use please in the normal course of explaining how to use a product, even if you're explaining a difficult task. Also don't use the phrase please note.

manishtr commented:
> @asm-foysal Thanks for the review.
> 
> Just one thing to share - It is kind of style issue, but using "please" is not recommended.
> 
> **Reference :** https://developers.google.com/style/word-list#letter-p
> 
> > Don't use please in the normal course of explaining how to use a product, even if you're explaining a difficult task. Also don't use the phrase please note.

Thanks for the review, i have replaced Please with Kindly

mijin85cho commented:
> > @asm-foysal Thanks for the review.
> > Just one thing to share - It is kind of style issue, but using "please" is not recommended.
> > **Reference :** https://developers.google.com/style/word-list#letter-p
> > > Don't use please in the normal course of explaining how to use a product, even if you're explaining a difficult task. Also don't use the phrase please note.
> 
> Thanks for the review, i have replaced Please with Kindly

Oh, maybe there was a misunderstanding. My comment was to 'remove' the "please", not to replace it.
To the developer guide, bald style is recommended..

Please remove the "Kindly"



**Reference :** https://developers.google.com/style/tone#politeness

manishtr commented:
> > > @asm-foysal Thanks for the review.
> > > Just one thing to share - It is kind of style issue, but using "please" is not recommended.
> > > **Reference :** https://developers.google.com/style/word-list#letter-p
> > > > Don't use please in the normal course of explaining how to use a product, even if you're explaining a difficult task. Also don't use the phrase please note.
> > 
> > 
> > Thanks for the review, i have replaced Please with Kindly
> 
> Oh, maybe there was a misunderstanding. My comment was to 'remove' the "please", not to replace it. To the developer guide, bald style is recommended..
> 
> Please remove the "Kindly"
> 
> **Reference :** https://developers.google.com/style/tone#politeness

I have removed "Kindly", Thanks for your review

mijin85cho commented:
@manishtr Thanks for the fast apply.

Hello, @asm-foysal 
As there were some minor updates, would you please approve again and merge this PR, if you see no more issue?

asm-foysal reviewed (approved):
@manishtr thanks for making the requested change.

@mijin85cho I gave it a quick review and everything is working great! Thank you for pointing out the issue about using the word "please".  I'm approving the PR for merging.

mijin85cho commented:
@asm-foysal Thanks for re-approving! Now you can merge this PR, to apply to docs.tizen.org.

mijin85cho commented:
As we have to publish now, I'll proceed instead of @asm-foysal
